index.html

```diff
@@ -5585,7 +5580,7 @@ <h3 id='fields'>Fields</h3>
 </tr>
 <tr>
 <td>use</td>
-<td>Complete (Bill ), Exploratory (Quote).</td>
+<td>Complete (Bill)</td>
```

Comment:
Same thing. Bill is an inaccurate term here.

Actually, it would be better to just remove the term text "(Bill)" entirely from here since we've hardcoded it to be Complete.

Reply:
Fixed.